### PR TITLE
KFSPTS-7318 moving DV and RCDV links to accounting section

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json [hunk 1]
```diff
@@ -35,6 +35,30 @@
...
+            "permission": {
+              "templateNamespace": "KR-SYS",
+              "templateName": "Initiate Document",
+              "details": {
+                "documentTypeName": "DV"
+              }
+            }
+          },
+          {
+            "label": "Recurring Disbursement Voucher",
```
Comment (bhutchinson) [inline src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json]: The only question, probably a functional one, I have is whether Recurring Disbursement Voucher should be right after the Disbursement Voucher link, or later in the list in alphabetical order.

Comment (jhulslander) [inline src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json]: I asked Cathy about the order, her response: "together..  put the DV in alphabetic order, have the recurring DV follow."

Comment (bhutchinson) [inline src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json]: Great, thanks!

**Comment by jhulslander** [comment]
thanks

**Pull request merged**